src/panoramic/cli/transform/commands.py

```diff
@@ -0,0 +1,89 @@
+from pathlib import Path
+from typing import List, Tuple
+
+import click
+from tqdm import tqdm
+
+from panoramic.cli.connections import Connections
+from panoramic.cli.context import get_company_slug
+from panoramic.cli.local.reader import FileReader
+from panoramic.cli.local.writer import FileWriter
+from panoramic.cli.paths import FileExtension, Paths
+from panoramic.cli.print import echo_error, echo_info
+from panoramic.cli.transform.executor import TransformExecutor
+from panoramic.cli.transform.pano_transform import PanoTransform
+
+
+def create_command():
+    echo_info('Scaffolding a new transform...')
+    name = click.prompt('name: ')
+
+    connections = Connections.load()
+    connection_names = connections.keys() if connections else []
+    connection_base_text = 'connection: '
+
+    if len(connection_names) == 0:
+        connection_prompt_text = connection_base_text
+    elif len(connection_names) > 3:
+        connection_prompt_text = f'{connection_base_text} (Available - {{{",".join(list(connection_names)[:3])}}},...)'
+    else:
+        connection_prompt_text = f'{connection_base_text} (Available - {{{",".join(connection_names)}}})'
+
+    # Assemble target based on input
+    connection = click.prompt(connection_prompt_text)
+
+    schema = click.prompt('schema:', default='pano')  # Fixme: think through a default schema?
+    view = click.prompt('view: ')
+    target = f'{connection}.{schema}.{view}'
+
+    transform = PanoTransform(name=name, fields=[], target=target)
```

Comment:
How about something like:

```
connection = click.prompt('connection: ')  # leaving out extra logic you got here for brevity
target = click.prompt(f'target: {connection}.')  # the user only fills in the missing part...
```

but tbh since connection isn't used I'd maybe just keep it simple and do:
```
target = click.prompt(f'target: ')
```

Otherwise we have to teach user that `schema` isn't just the schema but for snowflake it's actually `{database}.{schema}`

Reply:
Changed in f08e122ce60e39d9179e9631f26283c446184907 to prompt for connection as before and view path next. @kubamahnert are we okay with the naming?

Reply:
Further improved the prompts based on the conversation above in c1ca4267478fa397d5725a954be26a7d77b6b2ed.